models/yang/annotations/openconfig-network-instance-annot.yang

```diff
@@ -143,22 +124,2017 @@ module openconfig-network-instance-annot {
       }
     }
 
+    deviation /oc-netinst:network-instances/oc-netinst:network-instance/oc-netinst:protocols/oc-netinst:protocol/oc-netinst:bgp/oc-netinst:global {
+        deviate add {
+            sonic-ext:db-name "CONFIG_DB";
+            sonic-ext:table-name "BGP_GLOBALS";
+            sonic-ext:key-transformer "bgp_gbl_tbl_key_xfmr";
+        }
+    }
+
+    deviation /oc-netinst:network-instances/oc-netinst:network-instance/oc-netinst:protocols/oc-netinst:protocol/oc-netinst:bgp/oc-netinst:global/oc-netinst:config/oc-netinst:router-id {
+        deviate add {
+            sonic-ext:field-name "router_id";
+        }
+    }
+
+    deviation /oc-netinst:network-instances/oc-netinst:network-instance/oc-netinst:protocols/oc-netinst:protocol/oc-netinst:bgp/oc-netinst:global/oc-netinst:state/oc-netinst:router-id {
+      deviate add {
+            sonic-ext:field-name "router_id";
+      }
+    }
+
+    deviation /oc-netinst:network-instances/oc-netinst:network-instance/oc-netinst:protocols/oc-netinst:protocol/oc-netinst:bgp/oc-netinst:global/oc-netinst:config/oc-netinst:as {
+        deviate add {
+            sonic-ext:field-name "local_asn";
```

Comment:
why "local_asn" is marked as deviation ?

Reply:
sonic-ext:field-name is a hint to the transformer to modify the name of the OC attribute to the SONiC attribute.

Here the OC attribute is "as" whereas the SONiC attribute is "local_asn"